src/components/CIPipelineN/CIPipeline.tsx

```diff
@@ -280,140 +308,152 @@ export default function CIPipeline({
         return { index: stepsLength + 1, calculatedStageVariables: _inputVariablesListPerTask }
     }
 
-    // TODO: Move to async
-    const getInitialData = (): void => {
-        setPageState(ViewType.LOADING)
-        getSecurityModuleStatus()
-        if (ciPipelineId) {
-            // TODO: Would need to add loader for getAvailablePlugins to avoid state toggle
-            getInitDataWithCIPipeline(appId, ciPipelineId, true)
-                .then((ciResponse) => {
+    const validateStage = (stageName: string, _formData: PipelineFormType, formDataErrorObject?): void => {
```

Comment:
move these methods after the API call methods

Reply:
Done